### DISCUSSION TITLE
https://explorer.celo.org/mainnet is lagging

### DISCUSSION THREAD

**Discussion by silasbw** [celo-org/blockscout#928] [Q&A]
It occurs intermittently throughout the day and is observable in the "blocks" header.  E.g., you might see the last block was "12 minutes ago".  Is this a known issue and plans to address is?  It looks similar to https://github.com/celo-org/blockscout/discussions/920.

**Comment by VolodYgrun** [comment]
me too

**Comment by Vladislavv21** [comment]
and i!
